digicampipe/io/containers_mc.py

```diff
@@ -0,0 +1,518 @@
+"""
+Container structures for data that should be read or written to disk
+"""
```

Comment:
I wasn't sure, because io.containers_mc are in fact containers from ctapipe with addition of some containers specific for digicampipe. I don't know what is actual plan about merge digicampipe with ctapipe and, therefore, which way is better. That is why I prepared containers_mc which should be (I hope) compatible with both.

Reply:
I must admit .. I am totally lost in this discussion ... containers in ctapipe will be modified a lot ... so whatever we have here now .. will not help us much when merging I guess.
Let's just do what works...

Reply:
OK, so in that case I have made digicampipe.io.containers compatible with simtel_pipeline and we don't need containers_mc anymore. Please check if I haven't destroyed anything..

Reply:
I don't know how to check that, we still lack test for this I believe.